core/src/main/java/org/mule/runtime/core/event/mutator/RemovePropertyProcessor.java

```diff
@@ -4,19 +4,20 @@
  * license, a copy of which has been included with this distribution in the
  * LICENSE.txt file.
  */
-package org.mule.runtime.core.transformer.simple;
+package org.mule.runtime.core.event.mutator;
```

Comment:
Don't like this package name at all. Not sure it was 100% required to move them from transformer.simple, but obviously they are no longer true transformers so the change does make sense.  how about just in `event` or `core.processor.simple`?